scripts/ci/logcheck/check-restart-logs.sh

```diff
@@ -22,6 +22,7 @@ for logfile in "$@"; do
     fi
 done
 
+# TODO: revisit (update/delete) "compliance-previous" pattern after "ROX-26311" is resolved.
```

Comment:
```suggestion
# TODO(ROX-26311): revisit (update/delete) "compliance-previous" pattern after "ROX-26311" is resolved.
```
Let's mention in the ticket to revisit this comment once the ticket is resolved.